ovos_tts_plugin_server/__init__.py

```
@@ -94,6 +132,34 @@
+                "Content-Type": "application/json"
+            }
+            if apikey:
+                headers["Authorization"] = f"Bearer: {apikey}"
+            try:
+                url = f"{url}/v1/audio/speech"
+                params["input"] = sentence
+                data = json.dumps(params)
+                self.log.debug(f"Chosen OpenAI TTS server {url}")
+                self.log.debug(f"Request to OpenAI TTS server {data}")
+                r: requests.Response = requests.post(url=url, headers=headers, data=data,
+                                                    verify=self.verify_ssl,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)